packages/main/src/Table.js

```diff
@@ -569,6 +705,20 @@ class Table extends UI5Element {
 
 		return this._inViewport ? "absolute" : "sticky";
 	}
+
+	isInViewport() {
```

Comment:
Do you use "isInViewport" - it maybe left from merge conflicts  resolution, please double check